src/ipmanager/ui/views.py

```diff
@@ -54,3 +56,23 @@ def get_context_data(self, **kwargs):
       group = current_group
     )
     return context
+
+class CreateRelationView(CreateView):
+  form_class = RelationForm
+  template_name = 'ui/create_relation_form.html'
+  def get_success_url(self):
+    return reverse('single_group', args=[self.object.subject.key])
+
+  def get_context_data(self, **kwargs):
+    context = super().get_context_data(**kwargs)
+    current_group = Group.objects.filter(key=self.kwargs.get('key')).first()
```

Comment:
This can be placed inline in the `context.update()` statement below:

```python
context.update(
    group=Group.objects.filter(key=self.kwargs.get('key')).first(),
)
```